libs/eo/landing-page/shell/src/lib/eo-landing-page-header.component.ts

```diff
@@ -74,15 +80,33 @@ const selector = 'eo-landing-page-header';
       class="${selector}__toolbar watt-space-inset-squished-m"
     >
       <img src="assets/energyorigin-logo.svg" alt="EnergyOrigin" />
-      <a mat-button mat-flat-button routerLink="/login">Start</a>
+      <a
+        mat-button
+        mat-flat-button
+        *rxLet="loginUrl$ as loginUrl"
+        [href]="loginUrl"
+        >Start</a
+      >
     </mat-toolbar>
   `,
 })
-export class EoLandingPageHeaderComponent {}
+export class EoLandingPageHeaderComponent {
+  loginUrl$: Observable<string> = this.landingPageStore.authenticationUrl$.pipe(
+    map((response) => response)
```

Comment:
question: Why do we have a mapping operation that does nothing?

Reply:
This should do right? 

- loginUrl$: Observable<string> = this.landingPageStore.authenticationUrl$;